UnityProject/Assets/packages.config

```diff
@@ -1,4 +1,2 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<packages>
```

Comment:
this file is no longer needed if you uninstall NugetForUnity

Reply:
Didn't uninstall NugetForUnity , just removed Tomlyn  from

Reply:
NugetForUnity isn't needed anymore if you move tomlyn to the project itself